home_work/hw_11/bank_deposit/currency.py

```diff
@@ -0,0 +1,35 @@
+import requests
+from loguru import logger as log
+
+API_KEY = '1240d98b1bc1385eea7ad602'
```

Comment:
Please, do not store any secret(s) in the git repo\client side. It is better to use env variables\pass it when you run a program\ansible etc.
```suggestion
API_KEY = os.getenv('API_KEY')
```